bucket/rainmeter-portable.json

```diff
@@ -0,0 +1,55 @@
+{
+    "homepage": "https://rainmeter.net/",
+    "description": "Rainmeter is a desktop customization tool for Windows.",
+    "version": "4.3.0.3298",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/rainmeter/rainmeter/releases/download/v4.3.0.3298/Rainmeter-4.3.exe#/inst.exe",
+    "hash": "1bb57129f6871c7e5a835ded406de7096aebbe53196006ca0d00073948fbb16b",
+    "depends": "7zip",
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": "7z x \"$dir\\inst.exe\" -o\"$dir\" -aoa"
+            }
+        },
+        "32bit": {
+            "installer": {
+                "script": "7z x \"$dir\\inst.exe\" -o\"$dir\" -aos"
+            }
+        }
+    },
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\Rainmeter.ini\")) { New-Item \"$dir\\Rainmeter.ini\" -Type File | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\Rainmeter.data\")) { New-Item \"$dir\\Rainmeter.data\" -Type File | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\Rainmeter.stats\")) { New-Item \"$dir\\Rainmeter.stats\" -Type File | Out-Null }"
+    ],
+    "persist": [
+        "Rainmeter.ini",
+        "Rainmeter.data",
+        "Rainmeter.stats",
+        "Layouts",
+        "Skins",
+        "Plugins"
+    ],
+    "post_install": [
+        "if((Get-Content \"$dir\\Rainmeter.ini\") -eq $Null) { Copy-Item \"$dir\\Defaults\\Layouts\\illustro default\\Rainmeter.ini\", \"$dir\\Defaults\\Skins\", \"$dir\\Defaults\\Layouts\" \"$persist_dir\" -Force -Recurse | Out-Null }",
+        "if(Test-Path \"$dir\\Plugins.original\") { Copy-Item \"$dir\\Plugins.original\\*\" \"$persist_dir\\Plugins\" -Force -Recurse; Remove-Item \"$dir\\Plugins.original\" -Force -Recurse | Out-Null }",
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe.nsis\", \"$dir\\inst.exe\" -Force -Recurse"
+    ],
+    "bin": [
+        "Rainmeter.exe",
+        "SkinInstaller.exe"
```

Comment:
Is it really neede to shim SkinInstaller?? Does this have some command line handler?

Reply:
You can pass skin file to it, that's as far as I know, I just iterated on existing rainmeter.json.